README.md

```diff
@@ -47,12 +47,13 @@ end
   config.registration.unregister_on_halt = false
 ```
 
-### subscription-manager Configuration
+- **manager** select the registration manager implementation. By default the plugin will use the implementation for the subscription-manager's register command, you can however change that by setting the option to a different implementation:
```

Comment:
I am not sure about rewording to 'implementation', implementing this as a manager makes sense, but if this plugin will support other manager in future it's not just an implementation of the same thing.